_test_common/lib/test_phases.dart

```diff
@@ -96,7 +96,7 @@ Future<BuildResult> testBuilders(
   String logPerformanceDir,
   String expectedGeneratedDir,
 }) async {
-  packageGraph ??= buildPackageGraph({rootPackage('a'): []});
+  packageGraph ??= _packageGraphFromInputs(inputs.keys);
```

Comment:
Some tests call `testBuilders` without a custom package graph and `inputs` outside of the default `a` package. This caused test failures since we can't know which inputs are valid without looking at the package graph. I changed the default package graph to contain packages for all inputs. If this is not desirable I could also fix the offending tests instead.